src/pe/mod.rs

```diff
@@ -190,9 +243,25 @@ impl<'a> PE<'a> {
                     certificate_table.virtual_address,
                     certificate_table.size,
                 )?;
+
+                #[cfg(feature = "pe_authenticode")]
+                {
+                    let start = certificate_table.virtual_address as usize;
+                    let end = start + certificate_table.size as usize;
+                    excluded_sections.certtable = Some(start..end);
+                }
+            }
+
+            #[cfg(feature = "pe_authenticode")]
+            {
+                excluded_authenticode_sections = Some(excluded_sections);
             }
         }
         Ok(PE {
+            #[cfg(feature = "pe_source")]
```

Comment:
this does not need to be a cfg; PE already has lifetime tied to the bytes, so storing it in pe is fine (we do the same for mach-o).